ios/browser/web/brave_web_main_parts.mm

```diff
@@ -1,11 +1,11 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
we backdated it?

Reply:
This file was moved into the web subdirectory to match chromium, but its been around since 2019